internal/test/commands/test_commands.go

```diff
@@ -355,28 +355,50 @@ func (self *TestCommands) LineageTable() datatable.DataTable {
 
 // LocalBranches provides the names of all branches in the local repository,
 // ordered alphabetically.
-func (self *TestCommands) LocalBranches() (gitdomain.LocalBranchNames, error) {
-	output, err := self.QueryTrim("git", "branch")
+func (self *TestCommands) LocalBranches() (gitdomain.LocalBranchNames, gitdomain.LocalBranchNames, error) {
+	forEachRefFormat := strings.Join(
+		[]string{
+			"%(refname:lstrip=2)", // the branch name (without refs/heads/)
+
+			" ", // a literal space
```

Comment:
Brilliant idea to use literate programming style here to document the various parts of this complex string. A possible simplification is to use string concatenation:

```suggestion
	forEachRefFormat := "" +
		"%(refname:lstrip=2)" + // the branch name (without refs/heads/)

		" " + // a literal space
```

Reply:
I would prefer to keep the current approach. To me, `, //` looks nicer than `+ //`.

Reply:
No problem!